main.tf

```diff
@@ -1,10 +1,11 @@
 locals {
   APP_NAME = "workadventure-jitsi"
   APP_HOST = "jitsi.dev.dorf.world"
-  DOCKER_HOST_ADDRESS = null # Set this to the Internet routable IPv4 address of the node?
+  PUBLIC_IP_ADDRESS = "45.151.166.29"
   JICOFO_AUTH_USER = "focus"
   JVB_AUTH_USER = "jvb"
   JVB_BREWERY_MUC = "jvbbrewery"
+  JVB_PORT = "10000"
```

Comment:
please use a Terraform variable for the JVB port, with a default value of '10000'